diff -Naur hashable-1.2.3.1/Data/Hashable/Class.hs hashable-1.2.3.1-patch/Data/Hashable/Class.hs
--- hashable-1.2.3.1/Data/Hashable/Class.hs	2015-03-14 17:52:53.487217540 -0700
+++ hashable-1.2.3.1-patch/Data/Hashable/Class.hs	2015-03-14 17:52:32.501371710 -0700
@@ -451,7 +451,9 @@
 -- | Compute the hash of a TypeRep, in various GHC versions we can do this quickly.
 hashTypeRep :: TypeRep -> Int
 {-# INLINE hashTypeRep #-}
-#if __GLASGOW_HASKELL__ >= 702
+#if   __GLASGOW_HASKELL__ >= 710
+hashTypeRep (TypeRep (Fingerprint x _) _ _ _) = fromIntegral x
+#elif __GLASGOW_HASKELL__ >= 702
 -- Fingerprint is just the MD5, so taking any Int from it is fine
 hashTypeRep (TypeRep (Fingerprint x _) _ _) = fromIntegral x
 #elif __GLASGOW_HASKELL__ >= 606
