diff -Naur libmpd-0.9.0.1/libmpd.cabal libmpd-0.9.0.1-patch/libmpd.cabal
--- libmpd-0.9.0.1/libmpd.cabal	2015-02-02 20:07:38.010611170 -0800
+++ libmpd-0.9.0.1-patch/libmpd.cabal	2015-02-02 20:07:54.807501179 -0800
@@ -40,7 +40,6 @@
       , containers >= 0.3 && < 1
       , filepath >= 1 && < 2
       , mtl >= 2.0 && < 3
-      , old-locale >= 1 && < 2
       , text >= 0.11 && < 2
       , time >= 1.1 && < 2
 
@@ -108,7 +107,6 @@
       , containers
       , filepath
       , mtl
-      , old-locale
       , text
       , time
 
diff -Naur libmpd-0.9.0.1/src/Network/MPD/Commands/CurrentPlaylist.hs libmpd-0.9.0.1-patch/src/Network/MPD/Commands/CurrentPlaylist.hs
--- libmpd-0.9.0.1/src/Network/MPD/Commands/CurrentPlaylist.hs	2015-02-02 20:07:38.006611196 -0800
+++ libmpd-0.9.0.1-patch/src/Network/MPD/Commands/CurrentPlaylist.hs	2015-02-02 20:08:14.037171421 -0800
@@ -1,4 +1,4 @@
-{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE OverloadedStrings, FlexibleContexts #-}
 
 {- |
 Module      : Network.MPD.Commands.CurrentPlaylist
diff -Naur libmpd-0.9.0.1/src/Network/MPD/Util.hs libmpd-0.9.0.1-patch/src/Network/MPD/Util.hs
--- libmpd-0.9.0.1/src/Network/MPD/Util.hs	2015-02-02 20:07:38.004611209 -0800
+++ libmpd-0.9.0.1-patch/src/Network/MPD/Util.hs	2015-02-02 20:07:54.807501179 -0800
@@ -15,8 +15,7 @@
 
 import           Control.Arrow
 
-import           Data.Time.Format (ParseTime, parseTime, FormatTime, formatTime)
-import           System.Locale (defaultTimeLocale)
+import           Data.Time.Format (ParseTime, parseTime, FormatTime, formatTime, defaultTimeLocale)
 
 import qualified Prelude
 import           Prelude hiding        (break, take, drop, dropWhile, read)
diff -Naur libmpd-0.9.0.1/tests/StringConn.hs libmpd-0.9.0.1-patch/tests/StringConn.hs
--- libmpd-0.9.0.1/tests/StringConn.hs	2015-02-02 20:07:38.008611183 -0800
+++ libmpd-0.9.0.1-patch/tests/StringConn.hs	2015-02-02 20:18:32.031772414 -0800
@@ -39,7 +39,7 @@
     SMPD { runSMPD :: ErrorT MPDError
                       (StateT [(Expect, Response String)]
                        (ReaderT Password Identity)) a
-         } deriving (Functor, Monad, MonadError MPDError)
+         } deriving (Functor, Applicative, Monad, MonadError MPDError)
 
 instance MonadMPD StringMPD where
     getVersion  = error "StringConn.getVersion: undefined"
