diff -Naur xmonad-extras-0.12/XMonad/Actions/Volume.hs xmonad-extras-0.12-patch/XMonad/Actions/Volume.hs
--- xmonad-extras-0.12/XMonad/Actions/Volume.hs	2015-01-27 20:52:29.132858696 -0800
+++ xmonad-extras-0.12-patch/XMonad/Actions/Volume.hs	2015-01-27 20:47:15.050337622 -0800
@@ -52,6 +52,7 @@
     osdCat
 ) where

+import Prelude hiding ((<*))
 import Control.Monad
 import Control.Monad.Trans
 import Data.List.Split (splitOn)
